test/RockPaperScissors_test.cpp

```diff
@@ -1,5 +1,6 @@
 #include <gtest/gtest.h>
-
+#include "RockPaperScissors.h"
+using namespace std;
 class name : public ::testing::Test
```

Comment:
Like above, usually you want a single newline between functions, classes, and between your block of `#include`s and the rest of the file
```suggestion

class name : public ::testing::Test
```